### PR TITLE
Rename Thruna targets

### PR DESCRIPTION
**Pull request by gtaharaedmonds**
### Summary

Rename Thruna targets to avoid conflicts with Quadruna boards with the same name.

### Changelist 

- Rename Thruna targets in VS Code config, CMake, and CI

### Testing Done

- Basic testing in VS Code + CI

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [x] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

File: .github/actions/build_binary/action.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Build Firmware Binaries Action
-description: Builds the bianry for a given board
+description: Builds the binaryZ for a given board
```
Comment (will-chaba) [inline .github/actions/build_binary/action.yml:2]: typo

**Review by will-chaba — COMMENTED** [review]
Looks good to me, found a small typo

**Review by will-chaba — APPROVED** [review]

File: .github/actions/build_binary/action.yml (hunk 1 above)
Comment (Lucien950) [reply .github/actions/build_binary/action.yml:2]: 😨

Comment (Lucien950) [reply .github/actions/build_binary/action.yml:2]: i think the spelling looks just fine

**Review by Lucien950 — COMMENTED** [review]
consider this PR, officially, APPROVED
image

**Review by Lucien950 — APPROVED** [review]

**Comment by will-chaba** [comment]
> consider this PR, officially, APPROVED image

I think I need to up my approval message game

File: .github/actions/build_binary/action.yml (hunk 1 above)
Comment (gtaharaedmonds) [reply .github/actions/build_binary/action.yml]: whoops

**Pull request merged**